content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -156,11 +156,11 @@ kubeadm config images pull
 You can pass `--config` to the above commands with a [kubeadm configuration file](#config-file)
 to control the `kubernetesVersion` and `imageRepository` fields.
 
-All default `k8s.gcr.io` images that kubeadm requires support multiple architectures.
+All default `registry.k8s.io` images that kubeadm requires support multiple architectures.
 
 ### Using custom images {#custom-images}
 
-By default, kubeadm pulls images from `k8s.gcr.io`. If the
+By default, kubeadm pulls images from `registry.k8s.io`. If the
```

Comment:
Is this true?

Reply:
Did we backport this change to a patch release for v1.22? AIUI, v1.22 is out of support. We should take care how we communicate this.

Reply:
That's fair to not merge this since 1.22 is out of support. I was going by the versions who's docs are still available on the kubernetes.io dropdown. If the plan is to remove 1.22 soon then no issue closing this without the merge.

Reply:
If there's a kubeadm v1.22.17 that pulls from `registry.k8s.io` then this is fine to merge; otherwise, I think it's more likely to mislead.

https://www.downloadkubernetes.com/ doesn't go back that far and I don't _see_ a  v1.22.17 release anywhere, but maybe I missed something.

Reply:
I performed a test for kubeadm v1.22.17:

1. Run Ubuntu 22.04 Docker image (to get a clean env)
2. wget https://dl.k8s.io/v1.22.17/bin/linux/amd64/kubeadm
3.
   ``` 
   # ./kubeadm-v1.22.17 version -o short
   v1.22.17
   ```
4. 
   ```
   # ./kubeadm-v1.22.17 config images pull
   W1214 00:56:22.511218    3649 kubelet.go:213] cannot determine if systemd-resolved is active: no supported init system detected, skipping checking for services
   I1214 00:56:23.046180    3649 version.go:255] remote version is much newer: v1.26.0; falling back to: stable-1.22
   failed to pull image "registry.k8s.io/kube-apiserver:v1.22.17": output: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
   , error: exit status 1
   To see the stack trace of this error execute with --v=5 or higher
   ```
   We can see that kubeadm v1.22.17 tries to pull images from `registry.k8s.io` by default.

---

cf. for kubeadm v1.22.16:

```
# ./kubeadm-v1.22.16 version -o short
v1.22.16
```

```
# ./kubeadm-v1.22.16 config images pull
W1214 00:58:29.837328    3726 kubelet.go:213] cannot determine if systemd-resolved is active: no supported init system detected, skipping checking for services
I1214 00:58:30.404089    3726 version.go:255] remote version is much newer: v1.26.0; falling back to: stable-1.22
failed to pull image "k8s.gcr.io/kube-apiserver:v1.22.17": output: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
, error: exit status 1
To see the stack trace of this error execute with --v=5 or higher
```

We can see that kubeadm v1.22.16 tries to pull images from `k8s.gcr.io` by default.